client/views/join_channel.tpl

```diff
@@ -0,0 +1,5 @@
+<div id="join_channel_{{id}}" class="join_form">
+	<input type="text" class="input" id="joinchan_{{id}}:channel" name="channel" placeholder="#channel">
```

Comment:
You don't really need all these ids, since the event handler you can just do `$(this).find("input[name=...")` (or on classname if you add one)